packages/teleterm/src/ui/ModalsHost/ModalsHost.tsx

```diff
@@ -38,6 +39,16 @@ export default function ModalsHost() {
     );
   }
 
+  if (dialog.kind === 'cluster-remove') {
+    return (
+      <ClusterRemove
+        clusterUri={dialog.clusterUri}
+        clusterTitle={dialog.clusterTitle}
```

Comment:
seems that `serviceModals.closeDialog()` is repeated for every dialog, so lets add 
const handleClose = () => serviceModals.closeDialog() ?

Reply:
Done